bentoml/server/api_server.py

```diff
@@ -141,6 +165,7 @@ def static_serve(static_path, file_path):
             )
 
     @staticmethod
+    @log_traceback
```

Comment:
this is probably not necessary, it is only meant for wrapping around user-defined API callback function.

Reply:
no need to wrap it around static_serve nor index_view_func, I think @bojiang's suggestion makes sense. Using a decorator does not seem to make the code cleaner in this case